.github/workflows/android-maestro.yml

```
@@ -143,27 +168,37 @@ jobs:
                         set -x
-                        adb wait-for-device
+                        timeout 300 adb wait-for-device
                         timeout 180 bash -c 'until [ "$(adb shell getprop sys.boot_completed | tr -d "\r")" = "1" ]; do sleep 2; done'
                         (cd "${{ env.APP_DIR }}/android" && ./gradlew assembleRelease)
                         apk=$(find "${{ env.APP_DIR }}/android/app/build/outputs/apk/release" -name '*.apk' | head -1)
-                        adb install -r "$apk"
+                        timeout 300 adb install -r "$apk"
                         cd "${{ env.EXAMPLE_DIR }}"
                         yarn e2e:android:${{ matrix.target }}
 
             -   name: Capture final device state
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)